app/src/screens/SignIn/index.tsx

```diff
@@ -1,12 +1,13 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```

Comment:
Nitpick, I prefer us to do `React.useEffect` instead of importing `useEffect` itself.

Reply:
I don't have a preference but I'm curious, do you see issues importing useEffect itself?

Reply:
Agreed, what is the preference for using `React.` vs `useEffect` itself?

Reply:
No, but that's how it's done in other files. It's more for consistency's sake.